packages/cetlib-except/package.py

```diff
@@ -61,3 +61,7 @@ def setup_build_environment(self, env):
         env.prepend_path("PATH", os.path.join(self.build_directory, "bin"))
         # Cleanup.
         sanitize_environments(env, "PATH")
+
+    def setup_run_environment(self, run_env):
+        run_env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+        run_env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
`cetlib-except` provides no plugins, so this line should go away.

Also, `cetlib-except` does not depend upon ROOT, so I'm not sure updating `ROOT_INCLUDE_PATH` here is the right place.  I think it's probably better to just update `ROOT_INCLUDE_PATH` in `canvas-root-io` (where ROOT is first introduced), and it can be updated there to include `canvas-root-io`'s dependencies, which include `cetlib-except.